docs/modules/nifi/pages/usage_guide/external_ports.adoc

```diff
@@ -1,4 +1,4 @@
-= Adding External Files to the NiFi Servers
+= Expose ports fron NiFi processors outside of Kubernetes
```

Comment:
```suggestion
= Expose NiFi Processors Ports
```

Reply:
I thought we use sentence case for headings? https://docs.stackable.tech/home/nightly/contributor/style-guide.html#_tone_and_writing_style_google_developer_documentation_style_guide
 
I was so happy that I remembered when writing this :)

Also, happy with your change, but I think it needs an ' in that case?

> Expose NiFi processors' ports

Or maybe actually just 

> Expose NiFi processor ports